service/controller/kvm/v19/version_bundle.go

```diff
@@ -6,28 +6,6 @@ import (
 
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
-		Changelogs: []versionbundle.Changelog{
```

Comment:
Clean up changelogs

Reply:
There should be one changelog with `TODO` here